### DISCUSSION TITLE
[FR] Install profiles into SuperSlicer [Implemented]

### DISCUSSION THREAD

**Discussion by pandel** [sn4k3/UVtools#355] [Ideas]
Hi!

First, MANY MANY thanks for your great software! It is absolutely amazing!

My FR is only for convenience: if you find some time would it be possible to add an additional button to install printer profiles into SuperSlicer in addition to what we have with PrusaSlicer?

I tried it on my Mac, manually copied the profiles over from PrusaSlicer and they work flawlessly in SuperSlicer.

Regards
Holger

**Comment by sn4k3** [comment]
I never tried SuperSlicer.
If you have the user paths to copy over please post here

**Reply by pandel** [reply]
On Mac it is easy. Instead of using the base path

`~/Library/Application Support/PrusaSlicer`

it is

`~/Library/Application Support/SuperSlicer`

Don't know, if it is the same on Windows, but I don't think it is that different...

**Reply by sn4k3** [reply]
Ok, i will take a look.
What SuperSlicer offer more vs PrusaSlicer in SLA matter?

**Reply by sn4k3** [reply]
Implemented

**Reply by xPew** [reply]
Very welcomed button. I don't think it offers that much for SLA, but its also my go to for FDM. And I also own an SLA printer, so less app switching! :)

**Reply by pandel** [reply]
> Ok, i will take a look.
> What SuperSlicer offer more vs PrusaSlicer in SLA matter?

Oh sorry! Did not get any notice about your question... I also don't think, it has any additions regarding SLA, but it is absolutely amazing with FDM.
